doc/examples/features_detection/plot_corner.py

```diff
@@ -17,21 +17,23 @@
 from skimage.transform import warp, AffineTransform
 from skimage.draw import ellipse
 
-
+# Sheared checkerboard
 tform = AffineTransform(scale=(1.3, 1.1), rotation=1, shear=0.7,
-                        translation=(210, 50))
-image = warp(data.checkerboard(), tform.inverse, output_shape=(350, 350))
-rr, cc = ellipse(310, 175, 10, 100)
+                        translation=(110, 30))
+image = warp(data.checkerboard()[:99, :99], tform.inverse, output_shape=(200, 310))
```

Comment:
`:99, :99` looks a bit confusing, probably should change to `:100, :100` or similar. Also, I'd move `data.checkerboard()[:..., :...]` to a separate line.

Reply:
That was my first attemps @soupault but,
`In [10]: data.checkerboard()[:100,:100]                                                                                                                                                                                                      
Out[10]: 
array([[255, 255, 255, ...,   0,   0,  50],
       [255, 255, 255, ...,   0,   0,  50],
       [255, 255, 255, ...,   0,   0,  50],
       ...,
       [  0,   0,   0, ..., 255, 255, 205],
       [  0,   0,   0, ..., 255, 255, 205], 
       [ 50,  50,  50, ..., 205, 211, 175]], dtype=uint8)
`
you can see from this output the anti-aliasing effect that creates a quite disturbing line around the pattern.

Reply:
The data comes from a png picture, and they are not generated. Few options are possible:
* Option1: change the function checkerboard() to generate the data instead to reading a file.
* Option2: try to threshold the image in this example (I don't like it because it adds complexity for no gain).
* Option3: generate the checkerboard in this example.
* Option 4: leave it as it is.

@scikit-image/core Any opinion on this? I prefer option 1, then 3 and I do not like 2 at all.

If we apply option 1, I would like to do not block this issue

Reply:
Ah, yes, I see the issue. Maybe we just switch to the round number then to not overload the readers? 80 or 90 looks good.

Also, please, wrap at 80th column.

Reply:
@soupault Good suggestion. Done!